docs/faststore/docs/getting-started/1-onboarding/dashboard.mdx

```diff
@@ -76,41 +64,57 @@ This section lists all the deploy previews created for GitHub branches in your p
 
 ## Integrations
 
-The **Integrations** tab allows you to configure store pages for Lighthouse Tests. Here, you can specify the URLs for a product listing page (PLP) and a product details page (PDP) to undergo automatic Lighthouse score testing during deployments. This ensures a consistent testing experience.
+The **Integrations** tab allows you to configure store pages for Lighthouse Tests. Here, you can specify the URLs for a product listing page (PLP) and a product details page (PDP) to undergo automatic Lighthouse score testing during deployments, ensuring a consistent testing experience.
 
-![dashboard-integrations](https://vtexhelp.vtexassets.com/assets/docs/src/dashboard-integrations___8575a9f0fc2abfa64bdfc10fc7534e9d.png)
+![integrations-webops](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@EDU-14723-faststore-dashboard/images/integrations-webops.png)
 
 ### Configuring pages for Lighthouse testing
 
 To set the pages you want to test with Lighthouse, follow these steps:
 
-1. In the **Default Category for PLP** field, enter the [category](https://help.vtex.com/en/tutorial/what-is-a-category--6HV4Q3E2FauUoOQoiCCgCg) slug defined in the Catalog.
-2. In the **Default Product Slug for PDP**, enter the slug for the specific PDP you want to test.
+1. In the **Path for a PLP** field, enter the [category](https://help.vtex.com/en/tutorial/what-is-a-category--6HV4Q3E2FauUoOQoiCCgCg) slug defined in the Catalog.
+2. In the **Path for a PDP** field, enter the slug for the specific product detail page (PDP) you want to test.
 3. Click `Save`.
 
-Once you've set the pages for testing and after a new deploy, go to the **Overview** tab and access the **Lighthouse Scores** section to check the scores for those pages.
+Once you've set the pages for testing and after a new deployment, go to the **Overview** tab and access the **Lighthouse Scores** section to check the scores for those pages.
+
+## Settings
+
+The **Settings** tab allows you to quickly configure the production domains, without needing VTEX intervention.
+
+![settings-webops](https://cdn.jsdelivr.net/gh/vtexdocs/dev-portal-content@EDU-14723-faststore-dashboard/images/settings-webops.png)
+
+> ⚠️ The proper configuration of the **Settings** tab is an essential step in the process of going live. For further information, see [Configuring external DNS](https://developers.vtex.com/docs/guides/faststore/go-live-1-configuring-external-dns).
+### Production domain
+
+In the **Production domain** section, choose your store’s domain from the dropdown list within the **Host** field and then click `Add` to set up the primary domain of your store.
+### Current Domains
+
+In the **Current Domains** section are the default and the primary domains configured for your store.
+
+To verify if your domain is properly configured in the FastStore WebOps environment, navigate to the Settings tab and check if it appears in the **Current Domains** list.
```

Comment:
```suggestion
```

Reply:
Como essa seção é sobre o Current Domains, acho que vale explicar do ponto de vista de quem já está nela. 
- Para que ela serve? (This section serves as a reference point to ensure that your domains are correctly set up in WebOps.)
- Se um dominio esperado não estiver lá, o que isso significa? (If your domain is not listed, it may indicate an issue with the configuration or DNS settings that need to be addressed before proceeding with the go-live process.)